src/NHibernate/Type/TimeOnlyAsTicksType.cs

```diff
@@ -0,0 +1,54 @@
+#if NET6_0_OR_GREATER
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.Common;
+using System.Linq.Expressions;
+using System.Reflection;
+using NHibernate.Engine;
+using NHibernate.Hql.Ast;
+using NHibernate.Linq.Functions;
+using NHibernate.Linq.Visitors;
+using NHibernate.SqlTypes;
+using NHibernate.Util;
+
+namespace NHibernate.Type
+{
+
+	/// <summary>
```

Comment:
Nit: Undue blank line.


```suggestion
{
	/// <summary>
```